features/languages-testcase.feature

```diff
@@ -0,0 +1,39 @@
+         Feature: Languages
+
+Scenario Outline: User is able to change the language
+           Given User sees <country-flag> on the pah-fm website
+            When User clicks on <country-flag>
+            Then The language on the page changes to <chosen-language>
+
+        Examples:
+|    country-flag | chosen-language |
+|         Ukraine |       Ukrainian |
+|  United Kingdom |         English |
+|          Poland |          Polish |
+
+Scenario Outline: Language is retained after User login
+           Given User is on the login page
+             And User chooses <chosen-language> by clicking on a flag
+            When User log in <status>
```

Comment:
Which user logs in? I guess user preference should be kept in user profile (which implies that test will fail for two successful logins). Though I am not aware of business logic.

Reply:
I've checked with @adamgajzlerowicz and currently language is saved only in session